core/class/jMQTTCmd.class.php

```diff
@@ -246,13 +246,46 @@ public function preSave() {
 
         }
 
+		// Reset autoPub if info cmd (should not happen or be possible)
+		if ($this->getType() == 'info' && $this->getConfiguration('autoPub', 0))
+			$this->setConfiguration('autoPub', 0);
```

Comment:
would be better to remove it instead of set it to 0

Reply:
doesn't exist ...

Reply:
I think is should be set to null to be removed